enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,243 @@
+cluster relies on user provided configurations. 
+
+To improve the customers experience, this work proposes taking etcd backup using default config from day one, without additional configuration from the user.
+
+### Goals
+
+* Backups should be taken without configuration after cluster installation from day 1.
+* Backups are saved to a default PersistentVolume, that could be overridden by user.
+* Backups are taken according to a default schedule, that could be overridden by user.
+* Backups are taken according to a default retention policy, that could be overridden by user.
+
+### Non-Goals
+
+* Save cluster backups to remote cloud storage (e.g. S3 Bucket).
+  - This could be a future enhancement or extension to the API.
+  - Having backups in an independent storage from OCP cluster is crucial for Disaster Recovery scenarios.
+* Automate cluster restoration.
+* Provide automated backups for non-self hosted architectures like Hypershift.
+
+## Proposal
+
+To enable automated etcd backups of an Openshift cluster, a default etcd backup option to be introduced. 
+Using a `SideCar` container within each etcd pod, backups could be taken with no config from user.
+
+### User Stories
+
+* As a cluster administrator I would like OCP cluster backup to be taken without configuration.
+* As a cluster administrator I would like to schedule recurring OCP cluster backups so that I have a recent cluster state to recover from in the event of quorum loss (i.e. losing a majority of control-plane nodes).
+* As a cluster administrator I would like to have alerts upon failure to take OCP cluster backup.
+
+### API Extensions
+
+No [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) changes are required, since this approach work independently with default config.
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+TBD
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+### Workflow Description
+TBD
+
+## Design Details
+
+A `SideCar` container within each etcd pod in Openshift cluster is being added in order to take backups without any provided configuration from user's perspective.
+
+In order to alleviate the overhead on etcd server, the side container takes the backup by simply copying the snapshot file.
+This approach is being used by `microshift`, and it has minimal overhead as it is a simple file system copy operation.
+Since the `SideCar` is being deployed alongside each etcd cluster member, it is possible to keep backups across all master nodes.
+
+On the other hand, the backups may **not** be up-to-date since the snapshot might be lagging behind the `WAL`. Therefore, it is recommended to use this approach alongside the Automated Backup enabled using the `EtcdBackup` CR.
+Since this work will be enabled with no configuration, it is possible to use define a default values for the `Scheule`, `Retention` independently.
```

Comment:
It looks like this is completely different from what I remember we talked about last time. I'd like you to change the initial part of this document to clearly explain this mechanism is orthogonal to the user-configured backups.